demo/src/main/java/com/wishes/demo/configure/ApplicationConfig.java

```diff
@@ -40,6 +39,11 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
                 .addResourceLocations("/resources/");
```

Comment:
Can be configured using application.properties, no need to add it in java config:
_spring.mvc.static-path-pattern=/resources/**_
by default will be mapped to "static" folder of application